resharper/resharper-unity/src/ProjectModel/UnitySolutionTracker.cs

```diff
@@ -40,7 +40,8 @@ private void SetValues()
         {
             IsUnityProjectFolder.SetValue(HasUnityFileStructure(mySolution.SolutionDirectory));
             IsUnityProject.SetValue(IsUnityProjectFolder.Value && mySolution.IsValid() &&
-                                    mySolution.SolutionFilePath.ExtensionNoDot.ToLower() == "sln");
+                                    (mySolution.SolutionFilePath.ExtensionNoDot.ToLower() == "sln"
+                                    ||mySolution.SolutionFilePath.ExtensionNoDot.ToLower() == "slnf"));
```

Comment:
Is this check just to make sure we don't have a folder open? If so, can we simplify it to check that the solution isn't just a folder, and remove the explicit file extension check?